src/c++/perf_analyzer/model_parser.cc

```diff
@@ -265,6 +265,32 @@ ModelParser::InitTFServe(
   return cb::Error::Success;
 }
 
+cb::Error
+ModelParser::InitOpenAI(
+    const std::string& model_name, const std::string& model_version,
+    const int32_t batch_size)
+{
+  // OpenAI does not return model metadata hence we can not obtain any
+  // parameters.
+  model_name_ = model_name;
+  model_version_ = model_version;
+  max_batch_size_ = batch_size;
+
+  // OpenAI will take a single json input with a fully formed payload
+  auto in_it = inputs_->emplace("payload", ModelTensor()).first;
+  in_it->second.name_ = "payload";
```

Comment:
Is the only other place that needs to know this name/format the stimulus input data generation part that @nv-braf is working on?

Reply:
Yup! "Payload" is the magic key, and will be documented